src/dvc_render/vega.py

```diff
@@ -38,20 +46,40 @@ def __init__(self, datapoints: List, name: str, **properties):
             self.properties.get("template", None),
             self.properties.get("template_dir", None),
         )
+        self._optional_anchor_ranges: Dict[
+            str,
+            Union[
+                List[str],
+                List[List[int]],
+            ],
+        ] = {
+            "stroke_dash": [[1, 0], [8, 8], [8, 4], [4, 4], [4, 2], [2, 1], [1, 1]],
+            "color": [
+                "#945dd6",
+                "#13adc7",
+                "#f46837",
+                "#48bb78",
+                "#4299e1",
+                "#ed8936",
+                "#f56565",
+            ],
+            "shape": ["circle", "square", "triangle", "diamond"],
+        }
```

Comment:
I've updated but I do have mixed opinions on this approach. I'm not sure this is a great example but I have previously seen situations where constants led to tests passing where it would've been much more desirable for them to fail. I.e. I unintentionally changed the behaviour that someone else is relying on but the tests are setup so that they pass regardless.